packages/core/src/config/config.ts

```diff
@@ -89,9 +89,10 @@ export interface StoreCurrency {
     thousandsSeparator: string;
 }
 
-export type UserExperienceSettingNames = 'walletButtonsOnTop';
-
-export type UserExperienceSettings = { [key in UserExperienceSettingNames]: boolean };
+export interface UserExperienceSettings {
+    walletButtonsOnTop?: boolean;
```

Comment:
Don't think these should be optional.

Reply:
The `UserExperienceSettings` should be made optional instead?

Reply:
Does this make sense?
```
checkoutUserExperienceSettings: UserExperienceSettings | null;
```

Reply:
Think with the new change with moving `checkoutUserExperienceSettings` as part of global settings, we can safely remove null as well.

Reply:
Yes, I have updated this PR to reflect that `checkoutUserExperienceSettings` is always available now.